tests/render/test_column_section_renderer.py

```diff
@@ -74,6 +80,27 @@ def titanic_validation_results():
         return expectationSuiteValidationResultSchema.load(json.load(infile))
 
 
+@pytest.fixture
+def fake_expectation_with_description() -> Expectation:
+    class ExpectColumnAgesToBeLegalAdult(gxe.ExpectColumnValuesToBeBetween):
+        column: str = "ages"
+        min_value: int = 18
+        description: str = "column values must be a legal adult age"
+
+        @classmethod
+        @renderer(renderer_type="renderer.prescriptive")
+        @render_evaluation_parameter_string
+        def _prescriptive_renderer(
+            cls,
+            **_,
+        ):
+            assert (
+                False
+            ), "Expectation `description` should take precedence over `prescriptive_renderer`"
+
+    return ExpectColumnAgesToBeLegalAdult()
```

Comment:
I know we've been trying to avoid fake classes but `description` needs to be set on something and I don't want to modify source code